components/search/search.tsx

```diff
@@ -244,12 +296,18 @@ const Search: React.FC<Props> = (props: Props): JSX.Element => {
         if (indexChangedViaKeyPress && !searchType && !searchTerms) {
             e.preventDefault();
             setKeepInputFocused(true);
-            actions.updateSearchType(highlightedSearchHintIndex === 0 ? 'messages' : 'files');
-            setHighlightedSearchHintIndex(-1);
+            actions.updateSearchType(highlightedSearchItemIndex === 0 ? 'messages' : 'files');
+            setHighlightedSearchItemIndex(-1);
+            props.actions.getRecentSearches();
```

Comment:
If we get the recent search results on focus, I don't think we need to do it on enter as well since we'll update the list the next time the user focuses the search box

Reply:
So if the user hits the enter key on a recent search nothing should happen, right?

Reply:
Well, it'll still need to do whatever it did before this was added, but since we've already called this in `handleFocus` when the user clicked on the search box,, we don't really need to do it again now

Reply:
Oh right, my bad I misread the comment :+1: